Tests/AuthFoundationTests/KeychainTokenStorageTests.swift

```diff
@@ -17,7 +17,7 @@ import XCTest
 @testable import TestCommon
 
 final class KeychainTokenStorageTests: XCTestCase {
-    let mock = MockKeychain()
+    var mock: MockKeychain!
```

Comment:
No objection to this change, in the past we have done this to avoid long-lived variables as the test target may keep these values in memory for the life entire test suite (as opposed to class by class)

Reply:
Exactly the reason these test changes are being made.  XCTestCase instances last for the whole lifetime of the test run, so I'm just making it a common practice to make any test properties that maintain state to be made var, and setup and torn down for each invocation.